dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/data/upload/DataUploadRunnable.kt

```diff
@@ -27,20 +28,22 @@ internal class DataUploadRunnable(
     private val systemInfoProvider: SystemInfoProvider
 ) : UploadRunnable {
 
-    private var currentDelayInterval =
-        DEFAULT_DELAY
+    private var currentDelayInterval = DEFAULT_DELAY_MS
 
     //  region Runnable
 
     override fun run() {
         val batch = if (isNetworkAvailable() && isSystemReady()) {
             reader.readNextBatch()
         } else null
+
         if (batch != null) {
             consumeBatch(batch)
         } else {
-            delayTheRunnable()
```

Comment:
Just confirming. It was:
* `x`, `0.9x`, `0.81x`, ... when the batches were found sequentially (with the `min()` cap),
* jumping discretely to `x = 20s` on the first missing batch.

Now we do it smoothly:
* `x`, `0.9x`, `0.81x` on batch found (with the `min()` cap),
* `1.1x`, `1.21x`, ... on batch miss (with the `max()` cap),

right?

Reply:
For how it used to be, you're completely right, but on the way it's done now, to be more accurate it is : 
- `𝑥`, `0.9𝑥`, `0.81x`, … when the batch is found and processed successfully, that is when the upload returns 200, 4xx (indicating an invalid format or request) or an unknown error occurs (meaning the batch might be faulty)
- `𝑥`, `1.1𝑥`, `1.21𝑥`, … when there's no batch or the batch process fails, that is when the upload return 5xx, 403 (invalid or expired client token), or a network exception occurs (e.g.: DNS, Connection timeout, …)